Lib/asyncio/queues.py

```diff
@@ -214,6 +254,33 @@ async def join(self):
         if self._unfinished_tasks > 0:
             await self._finished.wait()
 
+    def shutdown(self, immediate=False):
+        """Shut-down the queue, making queue gets and puts raise
+        QueueShutDown.
```

Comment:
Nit: I like the "summary" part of a docstring to fit on one line.
```suggestion
        """Shut-down the queue, making queue gets and puts raise QueueShutDown.
```